Makefile

```diff
@@ -63,3 +63,145 @@ sign-release:
 release: build-release package-release sign-release
 	@echo "Release build complete for $(GOOS)/$(GOARCH)!"
 
+# Changelog extraction targets
```

Comment:
not necessary if you don't want to, but a small suggestion is to pull out the complicated sed statement, for example 

`CHANGELOG_PIPELINE = sed '$$d' | tail -n +2 | awk '/^### /{section=$$0; items=""; next} /^( *)?- /{items=items $$0 "\n"; next} /^$$/ && items{print section "\n" items; items=""}' | sed '/^$$/d'`

then do that

`@CHANGELOG_CONTENT=$$(sed -n '/## \[Unreleased\]/,/## \[/p' CHANGELOG.md | $(CHANGELOG_PIPELINE)); `